### PR TITLE
External secrets: add checker docs

### PR DESCRIPTION
**Pull request by Halamix2**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

#3944

### CODE REVIEW

File: development/tools/cmd/externalsecretschecker/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# External secrets checker
```
Comment (mmitoraj) [inline development/tools/cmd/externalsecretschecker/README.md:1]:
```suggestion
# External Secrets Checker
```

File: development/tools/cmd/externalsecretschecker/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Overview
+
+This command checks external secrets synchronization status, and if every secret have corresponding external secret.
```
Comment (mmitoraj) [inline development/tools/cmd/externalsecretschecker/README.md:5]:
```suggestion
This command checks external secrets synchronization status, and if every secret has a corresponding external secret.
```

File: development/tools/cmd/externalsecretschecker/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+go run main.go --kubeconfig=/Users/i542853/.kube/config --ignored-secrets "namespace/secretName,namespace/secretName2"
+```
+
+### Flags
+
+See the list of available flags:
+
+| Name                      | Required | Description                                                                                             |
+| :------------------------ | :------: | :------------------------------------------------------------------------------------------------------ |
+| **--kubeconfig**          |   Yes    | The path to the `kubeconfig` file, needed to connect to a cluster.                                      |
```
Comment (mmitoraj) [inline development/tools/cmd/externalsecretschecker/README.md:21]:
```suggestion
| **--kubeconfig**          |   Yes    | The path to the `kubeconfig` file needed to connect to a cluster.                                      |
```

File: development/tools/cmd/externalsecretschecker/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+| **--ignored-secrets**     |    No    | List of ignored secrets. List contains of secrets in `namespace/secretName` format, separated by comma. |
```
Comment (mmitoraj) [inline development/tools/cmd/externalsecretschecker/README.md:22]:
```suggestion
| **--ignored-secrets**     |    No    | The list of ignored secrets. The secrets are in the `namespace/secretName` format and are separated with commas. |
```

File: development/tools/cmd/externalsecretschecker/README.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+| **--namespaces**          |    No    | List of analyzed namespaces. The program scans all namespaces if the list is empty.                     |
```
Comment (mmitoraj) [inline development/tools/cmd/externalsecretschecker/README.md:23]:
```suggestion
| **--namespaces**          |    No    | The list of analyzed namespaces. The program scans all namespaces if the ignored secrets list is empty.                     |
```
Did you mean the ignored secrets list?
Just to make sure: Does the program scan all namespaces if the ignored secrets list is (totally) empty or all namespaces for which there aren't any ignored secrets on the list?

**Review by mmitoraj — COMMENTED** [review]
Please see my comments below. Thank you :)

File: development/tools/cmd/externalsecretschecker/README.md (hunk 5 above)
Comment (Halamix2) [reply development/tools/cmd/externalsecretschecker/README.md:23]: The program scans all namespaces unless a list of namespaces to scan is provided through `namespaces` flag.

File: development/tools/cmd/externalsecretschecker/README.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+### Flags
+
+See the list of available flags:
+
+| Name                      | Required | Description                                                                                                           |
+| :------------------------ | :------: | :-------------------------------------------------------------------------------------------------------------------- |
+| **--kubeconfig**          |   Yes    | The path to the `kubeconfig` file needed to connect to a cluster.                                                     |
+| **--ignored-secrets**     |    No    | The list of ignored secrets. The secrets are in the `namespace/secretName` format and are separated with commas.      |
+| **--namespaces**          |    No    | The list of analyzed namespaces separated by bomma. The program scans all namespaces if the namespaces list is empty. |
```
Comment (mmitoraj) [inline development/tools/cmd/externalsecretschecker/README.md:23]:
```suggestion
| **--namespaces**          |    No    | The list of analyzed Namespaces. The Namespaces names are separated with commas. The program scans all Namespaces if the Namespaces list is empty. |
```

File: development/tools/cmd/externalsecretschecker/README.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
+# External Secrets Checker
+
+## Overview
+
+This command checks external secrets synchronization status, and if every secret has a corresponding external secret.
```
Comment (mmitoraj) [inline development/tools/cmd/externalsecretschecker/README.md:5]:
```suggestion
This command checks external Secrets synchronization status, and if every Secret has a corresponding external Secret.
```

File: development/tools/cmd/externalsecretschecker/README.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+```
+
+### Flags
+
+See the list of available flags:
+
+| Name                      | Required | Description                                                                                                           |
+| :------------------------ | :------: | :-------------------------------------------------------------------------------------------------------------------- |
+| **--kubeconfig**          |   Yes    | The path to the `kubeconfig` file needed to connect to a cluster.                                                     |
+| **--ignored-secrets**     |    No    | The list of ignored secrets. The secrets are in the `namespace/secretName` format and are separated with commas.      |
```
Comment (mmitoraj) [inline development/tools/cmd/externalsecretschecker/README.md:22]:
```suggestion
| **--ignored-secrets**     |    No    | The list of ignored Secrets. The Secrets are in the `namespace/secretName` format and are separated with commas.      |
```

**Review by mmitoraj — COMMENTED** [review]
Namespaces and Secrets should be capitalized. Thank you.

**Review by mmitoraj — APPROVED** [review]

**Pull request merged**